library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundle.wdl

```diff
@@ -0,0 +1,80 @@
+task BuildCellRangerReference {
+  String ref_fasta
+  String gtf_file
+  String ref_name
+
+  String docker = "quay.io/humancellatlas/secondary-analysis-cellranger"
+  String memory = "416 GB"
+  Int boot_disk_size_gb = 12
+  String disk_space = "250"
```

Comment:
Would be better to use Int here and then use below like: `"local-disk ${disk} HDD"`.  Similarly for memory.

Reply:
will do!